src/cryptonote_core/service_node_list.h

```diff
@@ -562,19 +568,24 @@ class service_node_list {
         }
     }
 
-    /// Copies x25519 pubkeys (as strings) of all currently active SNs into the given output
-    /// iterator
+    /// Copies x25519 pubkeys (as strings) of all currently registered SNs into the given output
+    /// iterator.  (Before the feature::SN_PK_IS_ED25519 hardfork this only includes SNs with known
+    /// proofs.)
     template <std::output_iterator<std::string> OutputIt>
-    void copy_active_x25519_pubkeys(OutputIt out) const {
+    void copy_x25519_pubkeys(OutputIt out, cryptonote::network_type nettype) const {
```

Comment:
Minor, but the nettype can just be used from the class. I'll make a note to do that later.